src/js/core/factories/GridApi.js

```diff
@@ -249,16 +249,19 @@
             $rootScope.$emit.apply($rootScope, [eventId].concat(Array.prototype.slice.call(arguments)));
           };
 
+          var cleanup = [];
```

Comment:
This is basically the same thing as the destroySteps from above, right?  You may want to consider unifying their naming convention

Reply:
Yeah it's just scoped differently. I will start doing some real clean up an variable consistency type changes